### PR TITLE
EOL of midori

### PR DESCRIPTION
**Pull request by lkocman**
* Requested to be removed in 15.4
  https://lists.opensuse.org/archives/list/factory@lists.opensuse.org/message/Z5MVF7JBNQXKGNPT4TVML62NGHJSGWSI/

### CODE REVIEW

File: xml/release-notes.xml [hunk 1]
```diff
@@ -395,6 +395,12 @@
     Removed packages are not shipped as part of the distribution anymore.
    </para>
    <itemizedlist>
+    <listitem>
+     <para>
+       <package>midori</package> a lightweight web browser based on WebKit and GTK+ is no longer suppored
+       and is scheduued for removal in next release.
```
Comment (tux93) [inline xml/release-notes.xml:401]:
```suggestion
       and is scheduled for removal in next release.
```

**Review by tux93 — CHANGES_REQUESTED** [review]
Only a small typo there ;)

**Comment by KucharczykL** [comment]
This should be under Deprecated packages and features since it's still part of the OS.

File: xml/release-notes.xml (hunk 1 above)
Comment (lkocman) [reply xml/release-notes.xml]: Fixed in last commit.

**Comment by lkocman** [comment]
Should be fixed now.

**Review by tux93 — APPROVED** [review]
👍🏻

**Comment by KucharczykL** [comment]
Thank you!

**Pull request merged**